src/main/java/teammates/logic/core/InstructorsLogic.java

```diff
@@ -118,8 +119,9 @@ public InstructorAttributes getInstructorForRegistrationKey(String encryptedKey)
     }
 
     public List<InstructorAttributes> getInstructorsForCourse(String courseId) {
-
-        return instructorsDb.getInstructorsForCourse(courseId);
+        List<InstructorAttributes> instructorReturnList = instructorsDb.getInstructorsForCourse(courseId);
+        Collections.sort(instructorReturnList, InstructorAttributes.compareByName);
+        return instructorReturnList;
```

Comment:
Add a blank line before the return statement for consistency with the surrounding methods.